<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChainPowered</class>
 <widget class="QWidget" name="ChainPowered">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pivot A:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Vec3Setter" name="pivotA" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pivot B:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Vec3Setter" name="pivotB" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Basis A:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="RotationSetter" name="basisA" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Basis B:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="RotationSetter" name="basisB" native="true"/>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="switchBodies">
     <property name="text">
      <string>Switch Bodies</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Motor X:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="MotorSelector" name="motorX" native="true"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Motor Y:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="MotorSelector" name="motorY" native="true"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Motor Z:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="MotorSelector" name="motorZ" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RotationSetter</class>
   <extends>QWidget</extends>
   <header>rotationsetter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Vec3Setter</class>
   <extends>QWidget</extends>
   <header>vec3setter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MotorSelector</class>
   <extends>QWidget</extends>
   <header>motorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
